donkeycar/templates/cfg_complete.py

```diff
@@ -32,6 +32,25 @@
 CAMERA_FRAMERATE = DRIVE_LOOP_HZ
 CAMERA_VFLIP = False
 CAMERA_HFLIP = False
+# PI Camera specitic options, for more details see https://picamera.readthedocs.io/en/release-1.13/api_camera.html
```

Comment:
And here, instead of passing all camera parameters, you would add a single dictionary as camera parameters. This should be empty, here and then different users can overwrite this to their needs. This should also address @tikurahul comment.